.cbmc-batch/source/openssl/objects_override.c

```diff
@@ -24,6 +24,8 @@
  */
 int OBJ_txt2nid(const char *s) {
     // Currently these are the only values used in the ESDK
+    assert(!s || strcmp(s, "prime256v1") == 0 || strcmp(s, "secp384r1") == 0);
```

Comment:
This doubles the number of strcmps necessary, and doesn't actually catch anything the `assert(0)` below doesn't